drivers/regulator/regulator_i2c.c

```diff
@@ -176,6 +246,7 @@ static const struct regulator_driver_api api = {
 #define CONFIGURE_REGULATOR(id)									\
 	static struct regulator_data i2c_reg_##id##_data;					\
 	static struct regulator_config i2c_reg_##id##_cfg = {					\
+		.label = DT_INST_PROP(id, label),						\
```

Comment:
I don't think this is necessary. A devicetree label is used as the name field in a struct device by DEVICE_DT_INST_DEFINE already. See DEVICE_DT_NAME.

Reply:
Thanks for the heads up, I wasn't aware of the `name` field. I've removed the `label` field from the struct.